flare/otf.py

```diff
@@ -384,7 +387,7 @@ def update_positions(self, new_pos: 'ndarray'):
         else:
             self.structure.prev_positions = self.structure.positions
         self.structure.positions = new_pos
-        self.structure.wrap_positions()
+        self.structure.positions[:] = structure.wrap_positions()
```

Comment:
Good catch that wrap_positions doesn't actually update the wrapped_positions attribute of structure. But I think structure.wrap_positions() should be self.structure.wrap_positions() here? (I'm a bit surprised this didn't cause an error in the OTF unit test)

Reply:
Oh yeah you're right. I will double check the unit test here

Reply:
I notice in `otf.py` there's the `update_positions` function, which is never used. While in `md.py` there's another `update_positions` which is used in `otf.py`. It seems that they are doing different things. The one in `otf.py` has rescale_temperature. Do you know where to put it? @jonpvandermause

Reply:
addressed in commit b2fb7895bb906719302eaaab717786915a29f5aa